sqlglot/dialects/duckdb.py

```diff
@@ -662,6 +662,7 @@ class Generator(generator.Generator):
             exp.Levenshtein: unsupported_args("ins_cost", "del_cost", "sub_cost", "max_dist")(
                 rename_func("LEVENSHTEIN")
             ),
+            exp.DateBin: rename_func("TIME_BUCKET"),
```

Comment:
What about parsing DuckDB's `TIME_BUCKET` into `DateBin` too? You probably need an entry like `"TIME_BUCKET": exp.DateBin.from_arg_list` in `FUNCTIONS`.